README.md

```diff
@@ -37,6 +37,21 @@ r2env init
 r2env add radare2@git
 ```
 
+## Usage
+
+These are the first steps to use r2, read the book or find tutorials for more details
+
+```sh
+$ r2 /bin/ls   # open the binary in read-only mode
+> px 32        # print 32 byte hexdump current block
+> f~foo        # filter flags with ~grep (same as |grep)
```

Comment:
Having that it's for  the first steps, it's better to introduce jargon slowly, so better show example on function list, for example
```
$ r2 /bin/ls
> aaa           # analyze functions and cross references, same as `r2 -A`
> afl~sym    # filter functions with `~` ...

> oo+;w hello    # write `hello` in current offset
```
would be better to introduce `seek` in that list

Reply:
chk again, feel free to suggest more changes

Reply:
looks good. Actually I'm thinking about some tutorial being in code. Kind of vimtutor